stacks_queues/stack.py

```diff
@@ -21,18 +21,25 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        v = self.store.get_first()
+        self.store.remove_first()
+        return v
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.get_first() == None
 
     def __str__(self):
```

Comment:
✨ Alternatively, you could take advantage of the `LinkedList` class' `str` method